app/src/pages/inside/common/launchSuiteGrid/defectStatistics/donutChart/donutChart.scss

```diff
@@ -15,6 +15,11 @@
   line-height: 18px;
   word-break: keep-all;
 }
+.total-to-investigate {
+  background-color: $COLOR--to-investigate-counter;
+  color: $COLOR--white;
+  border-color: $COLOR--white !important;
```

Comment:
Why `!important`?

Reply:
because it is set as inline style depending on defect type color, red - for Product bug, etc.
see above in donutChart.jsx in line 130

there is a different style for To investigate counter 
![image](https://user-images.githubusercontent.com/3634380/56904225-892c2d80-6aa6-11e9-84c5-ffca7788e6eb.png)